src/domain/customscript-identity.ts

```diff
@@ -79,11 +73,10 @@ function asTaprootAddressInterface(
     confidentialAddress: extended.confidentialAddress,
     blindingPrivateKey: extended.blindingPrivateKey,
     derivationPath: extended.derivationPath,
-    taprootHashTree: extended.result.taprootHashTree,
-    taprootInternalKey: extended.result.taprootInternalKey,
```

Comment:
maybe this should be kept?

Reply:
or can be extracted from the descriptor?

Reply:
I would drop taproot members, the field `contract` (Ionio contract) should be enough to compute "taproot witness stack" for signing step

Reply:
right, but Ionio Artifacts currently does not expose the internal PublicKey used in the compilation of the script.

But point taken, will expose a new field in the artifact then to retrieve it from there https://github.com/ionio-lang/ionio/issues/13

Reply:
These fields have been removed just from the type returned to the user, they should be cached by the identity still.